containers/proxy-helm/templates/_helpers.tpl

```diff
@@ -0,0 +1,11 @@
+{{- define "deployment.container.image" -}}
+{{- $imageName := .name -}}
+{{- $uri := (printf "%s/%s:%s" .global.Values.defaultRegistry $imageName .global.Values.defaultVersion) | default .global.Chart.AppVersion -}}
```

Comment:
Shouldn't the `.global.Chart.AppVersion` be ORed with the `.global.Values.defaultVersion` instead of the whole value? The question also applies for the other images